database/migrations/2023_06_12_170126_add_fields_to_asset_maintenance_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddFieldsToAssetMaintenanceTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('asset_maintenances', function (Blueprint $table) {
+            $table->integer('assigned_to')->after('user_id')->default(null)->nullable();
+            $table->integer('repairer')->after('assigned_to')->default(null)->nullable();
+            $table->integer('asset_maintenance_location')->after('asset_maintenance_type')->default(null)->nullable();
```

Comment:
My concern with adding this is that it could cause some confusion for people who don't do in-house repairs.

Reply:
Would an '(optional)' label be helpful here then?

Reply:
I can remove `repairer` in line with the though of the comment above. and put optional tags on the maintenance form?

Reply:
I can't speak for how others would use this but my use-case is to keep track of / run reports of users/locations who, based on trend seem to be damaging their devices more often than others and for appropriate costs per cost centre.

Depending on how long the asset is being repaired for, they may be assigned a new device and the originally damaged device put in stock once repaired.
It doesn't matter too much to us what the columns are called as long as we are able to select a user/location ourselves within the maintenance and this sticks to the maintenance rather than changing based on the assets properties (which is how location works at the moment)